cms/static/cms/js/require-config.js

```diff
@@ -1,289 +1,315 @@
+                deps: ["jquery"],
+                exports: "jQuery.ui"
+            },
+            "jquery.form": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.ajaxForm"
+            },
+            "jquery.markitup": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.markitup"
+            },
+            "jquery.leanmodal": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.leanModal"
+            },
+            "jquery.ajaxQueue": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.ajaxQueue"
+            },
+            "jquery.smoothScroll": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.smoothScroll"
+            },
+            "jquery.cookie": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.cookie"
+            },
+            "jquery.qtip": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.qtip"
+            },
+            "jquery.scrollTo": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.scrollTo",
+            },
+            "jquery.flot": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.plot"
+            },
+            "jquery.fileupload": {
+                deps: ["jquery.ui", "jquery.iframe-transport"],
+                exports: "jQuery.fn.fileupload"
+            },
+            "jquery.fileupload-process": {
+                deps: ["jquery.fileupload"]
+            },
+            "jquery.fileupload-validate": {
+                deps: ["jquery.fileupload"]
+            },
+            "jquery.inputnumber": {
+                deps: ["jquery"],
+                exports: "jQuery.fn.inputNumber"
+            },
+            "jquery.tinymce": {
+                deps: ["jquery", "tinymce"],
+                exports: "jQuery.fn.tinymce"
+            },
+            "datepair": {
+                deps: ["jquery.ui", "jquery.timepicker"]
+            },
+            "underscore": {
+                exports: "_"
+            },
+            "backbone": {
+                deps: ["underscore", "jquery"],
+                exports: "Backbone"
+            },
+            "backbone.associations": {
+                deps: ["backbone"],
+                exports: "Backbone.Associations"
+            },
+            "backbone.paginator": {
+                deps: ["backbone"],
+                exports: "Backbone.Paginator"
+            },
+            "youtube": {
+                exports: "YT"
+            },
+            "codemirror": {
+                exports: "CodeMirror"
+            },
+            "codemirror/stex": {
+                deps: ["codemirror"]
+            },
+            "tinymce": {
+                exports: "tinymce"
+            },
+            "mathjax": {
+                exports: "MathJax",
+                init: function() {
+                window.MathJax.Hub.Config({
```

Comment:
Bad indenting in here. After looking at the code more, I think the "window" is fine to have (though not necessary).